src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -41,6 +41,13 @@ internal static class CompletionOptions
             new PerLanguageOption<bool>(nameof(CompletionOptions), nameof(ShowNameSuggestions), defaultValue: true,
             storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ShowNameSuggestions"));
 
+        //Dev16 options
+
+        // Use tri-value so the default state can be used to turn on the feature with experimentation service.
+        public static readonly PerLanguageOption<bool?> ShowImportCompletionItems =
+            new PerLanguageOption<bool?>(nameof(CompletionOptions), nameof(ShowImportCompletionItems), defaultValue: null,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ShowImportCompletionItems"));
```

Comment:
@jasonmalinowski is okay for us to create new key like this? 

@genlu is this option temporary? or a permanent one?

Reply:
This is likely to change in the future, if we get the new VS shortcut for extended list.

Reply:
@jasonmalinowski Right now the option is basically "enable/disable this feature", since we don't have the extend-list trigger from editor team ready yet, we can only dump everything to the basic completion list. Once we have the new trigger available, I image this would become "in addition to show import item in extended list, also show them in basic completion list"

THis is the extended list I'm talking about
https://devdiv.visualstudio.com/DevDiv/_git/d9dc88ef-476f-4add-967b-f7ddd47f4b71/pullrequest/172231?_a=overview

Reply:
Then you may wish to consider appropriate renames *now* to something like "Show types from unimported namespaces in basic list". Which is conveniently also true in that in this current form it's _also_ showing it in just the basic list. (That's because the extended list doesn't exist yet!)